source/DasBlog All.sln

```diff
@@ -95,3 +95,15 @@ Global
 		SolutionGuid = {8F3ACFB8-CB52-4953-A1CF-4151E2789889}
 	EndGlobalSection
 EndGlobal
+
```

Comment:
Hi @poppastring, 

Yes, I used VS code. I believe I got a little confuse regarding the section 2 of the contributing file, when it says to open the DasBlog All.sln and perform a build. I will transfer the code to the right file and perform another PR, okay? 

I'm really sorry for the trouble.

Reply:
No, worries! This is good opportunity to learn something new. :) 
For this kind of project Visual Studio is the best option (VS Community is free) rather than VS Code.